### DISCUSSION TITLE
struggling with warping edges after arc overhangs?

### DISCUSSION THREAD

**Discussion by sanczess** [nicolai-wachenschwan/arc-overhang-prusaslicer-integration#7] [Ideas]
Arc overhangs are an amazing thing when it comes to flat overhangs, which before only could be printed with uggly supports.
However we see that following layers after arc overhangs tend to warp.
Heres one idea to solve that. 
If you merge two features arc overhangs and pin support you could get rid of warping.
So the Idea is to print a part as 5mm in height then add a pin suppurt underneath a later created edge of arc overhangs. And keep doing this each 5mm of z- height.
In the end the pin support and the arc overhang will stick together and prevent warping after printing following layers.

**Comment by nicolai-wachenschwan** [comment]
Hey Sanczess,
thank you for your submitted Idea! We could use the PrusaSlicer 2.6 tree supports for that, so we don't need to code anything, since evaluating if a structure needs support is kinda messy process for computers to do...
